scss/_mixins.scss

```diff
@@ -4,3 +4,4 @@
 @import "./mixins/states";
 @import "./mixins/type";
 @import "./mixins/vars";
+@import "./mixins/breakpoints";
```

Comment:
breakpoints is unrelated. either remove, or retitle PR

Reply:
@bcullman the breakpoints file is utilized on line 14 of `_display.scss` with the `breakpoint-infix` function.

Reply:
@bcullman ~ so they come hand in hand. they were deleted together.

Reply:
then maybe the PR should be generalize of reverting removed classes and mixin

Reply:
@droshev @bcullman ~ changed title of PR to `fix: restores display helpers with relevant mixins`